### PR TITLE
Updated all Linux distros, macOS and Docker

### PR DESCRIPTION
**Pull request by sashwathn**
I have updated all the Linux distros (currently listed), macOS, Docker, Kubernetes and Windows.
This is a quick PR to check how the Deploy sections will be rendered with this content.

Please check and approve.

### CODE REVIEW

File: integrations/deploy.yaml [hunk 1]
```diff
@@ -43,8 +43,228 @@
...
+  meta:
+    name: MacOS
+    link: ''
+    categories:
+      - deploy.operating-systems
+    icon_filename: ''
+  keywords:
+    - MacOS
+    - mac
+  description: 'Run the following command on your OSX, macOS server to install and claim Netdata:"r'
```
Comment (shyamvalsan) [inline integrations/deploy.yaml:182]: I saw in @Ferroin PR that this is only supported for Intel based Macs, if this is the case shouldn't we mention this here as well?

File: integrations/deploy.yaml [hunk 2]
```diff
@@ -43,8 +43,228 @@
...
+            --security-opt apparmor=unconfined \
+            netdata/netdata:stable \
+            -e NETDATA_CLAIM_TOKEN= {% claim_token %} \
+            -e NETDATA_CLAIM_URL={% claim_url %}
+  additional_info: ''
+  related_resources: {}
+  platform_info:
+    group: 'no_include'
+    distro: 'docker'
+  quick_start: 1
```
Comment (shyamvalsan) [inline integrations/deploy.yaml:270]: What about Windows?

**Review by shyamvalsan — CHANGES_REQUESTED** [review]

**Review by andrewm4894 — DISMISSED** [review]

File: integrations/deploy.yaml (hunk 2 above)
Comment (sashwathn) [reply integrations/deploy.yaml:410]: @shyamvalsan : I will need your help in updating Windows.. I have now updated all the others.

Comment (shyamvalsan) [reply integrations/deploy.yaml:410]:
```
1. Install [Windows Exporter](https://github.com/prometheus-community/windows_exporter) on every Windows host you want to monitor.
2. Install Netdata agent on Linux, FreeBSD or Mac.
3. Configure Netdata to collect data remotely from your Windows hosts by adding one job per host to windows.conf file. See the [configuration section](https://learn.netdata.cloud/docs/data-collection/monitor-anything/System%20Metrics/Windows-machines#configuration) for details.
4. Enable virtual nodes configuration so the windows nodes are displayed as separate nodes.
```

**Review by shyamvalsan — APPROVED** [review]

**Review by shyamvalsan — APPROVED** [review]

**Pull request merged**